addons/web/static/tests/legacy/helpers/mock_server.js

```diff
@@ -845,9 +845,21 @@ export class MockServer {
         const { records } = this.models[model];
         const result = [];
         for (const r of records) {
+            const extra = {};
+            if (kwargs.extra_fields) {
+                for (const field of kwargs.extra_fields) {
+                    if (r[field]) {
+                        extra[field] = r[field];
+                    }
```

Comment:
only send the value if it is truthy? Why do you need the if?

Reply:
Also, it means you could have extra for some records, and not for others, even though you asked for it. This doesn't look clean. I would always return a triple [id, display_name, extra] if extra fields are requested